### ISSUE TITLE
show a mean squared average between epochs like pyFM?

### ISSUE THREAD

**Issue by thestick613** [geffy/tffm#19]
https://github.com/coreylynch/pyFM#getting-started

**Comment by geffy** [comment]
I don't think that it's a really useful feature, but you can can set `verbose=2` and get avarage trainig loss printed, like this:
```python
from tffm import TFFMClassifier
model = TFFMClassifier(order=2, rank=10,  n_epochs=5, verbose=2)
model.fit(X_tr, y_tr)
```
```
[epoch 0]: mean target value: 0.6933335065841675
[epoch 1]: mean target value: 0.599380612373352
[epoch 2]: mean target value: 0.4427681267261505
[epoch 3]: mean target value: 0.38958144187927246
[epoch 4]: mean target value: 0.3296085000038147
```
